packages/playwright-core/bin/reinstall_chrome_beta_mac.sh

```diff
@@ -6,7 +6,7 @@ rm -rf "/Applications/Google Chrome Beta.app"
 cd /tmp
 curl -o ./googlechromebeta.dmg -k https://dl.google.com/chrome/mac/universal/beta/googlechromebeta.dmg
 hdiutil attach -nobrowse -quiet -noautofsck -noautoopen -mountpoint /Volumes/googlechromebeta.dmg ./googlechromebeta.dmg
-cp -rf "/Volumes/googlechromebeta.dmg/Google Chrome Beta.app" /Applications
+cp -pR "/Volumes/googlechromebeta.dmg/Google Chrome Beta.app" /Applications
```

Comment:
Should we use `ditto` instead? Did you manage to reproduce the issue?

Reply:
Yes was able to reproduce. Both will work, I’d vote for cp, since its more gnu like.